packages/components/time_region_value/src/node_modules/components/SettingsView.svelte

```diff
@@ -0,0 +1,95 @@
+<script>
+	import Switch from '@svizzle/ui/src/Switch.svelte';
+
+	import RegionsSelector from 'components/RegionsSelector.svelte';
+
+	const defaultHandlers = {
+		toggledColorScheme: null,
+		toggledFiltering: null,
+		toggledRanking: null,
+	}
+
+	export let colorScheme = {
+		value: 'Red-Blue',
+		values: ['Red-Blue', 'Green-Blue']
+	}
+	export let flags = {
+		areThereUnselectedNUTS1Regions: false,
+		doFilter: false,
+		showRankingControl: false,
+	}
+	export let handlers = defaultHandlers;
+
+	$: handlers = handlers ? {...defaultHandlers, ...handlers} : defaultHandlers;
```

Comment:
Could maybe use separate lines:
```
	$: handlers = handlers
		? {...defaultHandlers, ...handlers}
		: defaultHandlers;
```

Reply:
Not sure that it's needed as the line is shorter than 80 chars.